services/minespace-web/src/tests/components/Forms/projects/majorMineApplication/MajorMineApplicationForm.spec.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import MajorMineApplicationForm from "@/components/Forms/projects/majorMineApplication/MajorMineApplicationForm";
+import * as MOCK from "@mds/common/tests/mocks/dataMocks";
+import * as FORM from "@/constants/forms";
+import { reduxForm } from "redux-form";
+import { ReduxWrapper as CommonReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
+import { ReduxWrapper as MinespaceReduxWrapper } from "@/tests/utils/ReduxWrapper";
+import { BrowserRouter } from "react-router-dom";
+
+import { useSelector } from "react-redux";
+
+const mockDispatch = jest.fn();
+
+jest.mock("@mds/common/providers/featureFlags/useFeatureFlag", () => ({
+  useFeatureFlag: () => ({
```

Comment:
I'm just curious if this is necessary- we have it mocked in all 3 setupTests files so it shouldn't be necessary if that's working correctly.